.github/workflows/main.yml

```diff
@@ -0,0 +1,165 @@
+        uses: actions/checkout@v2
+        with:
+          repository: googleapis/discovery-artifact-manager
+          path: ./discovery-artifact-manager
+
+      - name: Copy scripts
+        run: |
+          cp ../branch_with_scripts/scripts/* .
+        working-directory: ./branch
+
+      - name: Copy discovery artifacts
+        run: |
+          cp ../discovery-artifact-manager/discoveries/*.json googleapiclient/discovery_cache/documents/
+        working-directory: ./branch
+
+      - name: Get changed discovery artifacts
+        run: git diff origin/master --name-only -- '*.json' |
+              sed 's/.*\///' > changed_files
+        working-directory: ./branch
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+
+      - name: Install dependencies
+        run: pip3 install -r requirements.txt
+        working-directory: ./branch
+
+      - name: Install google-api-python-client
+        run: pip3 install -e .
+        working-directory: ./branch
+
+      # Apply a workaround to discovery.py to avoid DefaultCredentialsError
+      # which is raised when calling `build_from_document()` as a result of
+      # `google.auth.default()`. The workaround is to bypass the code block that
+      # attempts to load default credentials.
+      - name: Workaround to avoid DefaultCredentialsError in discovery.py
+        run: sed -i -e 's/if credentials is None/if False/g' googleapiclient/discovery.py
+        working-directory: ./branch
+
+      - name: Update the docs
+        run: python3 describe.py
+        working-directory: ./branch
+
+      - name: Run Change Summary
+        run: python3 changesummary.py
+        working-directory: ./branch
+
+      - name: Commit changes
+        run: ./createcommits.sh
+        working-directory: ./branch
+
+      - name: Push changes
+        run: git push -f --set-upstream origin update-discovery-artifacts
+        working-directory: ./branch
+
+      - name: Prepare summary for PR Body
+        id: pr_body
+        shell: bash
+        run: |
+          python3 buildprbody.py
+          output=$(cat temp/allapis.summary)
+          output="${output//'%'/'%25'}"
+          output="${output//$'\n'/'%0A'}"
+          output="${output//$'\r'/'%0D'}"
+          echo "::set-output name=change_summary::$output"
+        working-directory: ./branch
+
+      - name: Create PR
+        uses: actions/github-script@v2.0.0
+        with:
+          github-token: ${{secrets.YOSHI_CODE_BOT_TOKEN}}
+          script: |
+            async function createPR () {
+              const { owner, repo } = context.repo
+              const branch = 'update-discovery-artifacts'
+              let prBody = `${{ steps.pr_body.outputs.change_summary }}`
+              const prTitle = 'chore: Update discovery artifacts'
+              const pullRequests = await github.pulls.list({
+                owner: owner,
```

Comment:
Formatting looks much better now 👍

Reply:
Great! Thanks for introducing me to `npx`!